src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -88,7 +151,7 @@ internal bool CanCreateModel(MutableObjectBinderContext context)
             // recursion.
             //
             // First, we want to make sure this object is allowed to come from a value provider source as this binder
-            // will always include value provider data. For instance if the model is marked with [FromBody], then we
+            // will only include value provider data. For instance if the model is marked with [FromBody], then we
```

Comment:
Changed ALL to any. If you want any more specific cleanup on the parts of the code that aren't changing I suggest you just do it and check it in :trollface: